TypeScript.py

```diff
@@ -281,6 +281,9 @@ def __init__(self):
         self.fileMap = {}
         self.refInfo = None
         self.versionST2 = False
+        self.tempFileMap = {}
+        self.tempFileList = []
```

Comment:
Cool, that makes much sense. To make it more clear, could you rename the ``tempFileList`` to something like ``available_tempfile_list``?